lib/core/src/Cardano/Wallet/Api/Link.hs

```diff
@@ -633,6 +634,21 @@ getTransaction w t = discriminate @style
     tid = ApiTxId (t ^. typed @(ApiT (Hash "Tx")))
     mkURL mk = mk wid tid
 
+createUnsignedTransaction
+    :: forall style w.
+        ( HasCallStack
+        , HasType (ApiT WalletId) w
+        , Discriminate style
+        )
+    => w
+    -> (Method, Text)
+createUnsignedTransaction w = discriminate @style
+    (endpoint @(Api.ConstructTransaction Net) (wid &))
+    (notSupported "Byron")
+    (notSupported "Shared")
```

Comment:
Perhaps add a `TODO: ADP-909` comment or something - These wallet types should be supported in the final version.